tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -167,17 +189,20 @@ def assert_sent_header_ok(cls):
         assert message_id == "999999123"
 
     @classmethod
-    def make_response(cls, score, include_score, include_status, status_code):
+    def make_response(
+        cls, score, include_score, include_status, status_code, include_description
+    ):
         header_info = {"imsx_version": "V1.0", "imsx_messageIdentifier": 1313355158804}
 
         if include_status:
             header_info["imsx_statusInfo"] = {
                 "imsx_codeMajor": status_code,
                 "imsx_severity": "status",
-                "imsx_description": "Result read",
                 "imsx_messageRefIdentifier": "999999123",
                 "imsx_operationRefIdentifier": "readResult",
             }
+            if include_description:
+                header_info["imsx_statusInfo"]["imsx_description"] = "Result read"
```

Comment:
The "Result read" description implies success, although we're actually interested in this for error cases. It doesn't matter for correctness, but it might help for documentation to use a different description when the status is an error.

Reply:
yep!